pandas/tests/io/parser/test_parsers.py

```diff
@@ -3,8 +3,10 @@
 import os
 import pandas.util.testing as tm
 
-from pandas import read_csv, read_table
+from pandas import read_csv, read_table, DataFrame, Index
```

Comment:
Looks like the `Index` import isn't used: https://travis-ci.org/pandas-dev/pandas/jobs/270584864#L1803